pylutron/__init__.py

```diff
@@ -43,7 +43,7 @@ class LutronConnection(threading.Thread):
   """Encapsulates the connection to the Lutron controller."""
   USER_PROMPT = b'login: '
   PW_PROMPT = b'password: '
-  PROMPT = b'GNET> '
+  PROMPT = b'> '
```

Comment:
Ah, I see you just pulled #22  here. We should probably jsut fix that one up, merge that, and you rebase on top of it.

Reply:
Yes, I've pulled #22 because I wasn't sure why it was stuck. Just let me know